docs/testplan/DPU-test-plan.md

```diff
@@ -0,0 +1,863 @@
+ * Use command `dpulink.sh` to assing ip address
+ * Use command `echo 1 > /sys/bus/pci/rescan` to rescan pcie links
+
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch: Showing example of one dpu pcie link
+
+root@sonic:/home/cisco# echo 1 > /sys/bus/pci/devices/0000:1a:00.0/remove
+root@sonic:/home/cisco# ping 169.254.28.1
+PING 169.254.28.1 (169.254.28.1) 56(84) bytes of data.
+^C
+--- 169.254.28.1 ping statistics ---
+2 packets transmitted, 0 received, 100% packet loss, time 1012ms
+
+root@sonic:/home/cisco# 
+root@sonic:/home/cisco# 
+root@sonic:/home/cisco# echo 1 > /sys/bus/pci/rescan
+root@sonic:/home/cisco# dpulink.sh
+Hello Mt Fuji
+169.254.24.1
+169.254.28.1
+169.254.32.1
+169.254.36.1
+169.254.139.1
+169.254.143.1
+169.254.147.1
+169.254.151.1
+root@sonic:/home/cisco# ping 169.254.28.1
+PING 169.254.28.1 (169.254.28.1) 56(84) bytes of data.
+64 bytes from 169.254.28.1: icmp_seq=1 ttl=64 time=0.329 ms
+^C
+--- 169.254.28.1 ping statistics ---
+1 packets transmitted, 1 received, 0% packet loss, time 0ms
+rtt min/avg/max/mdev = 0.329/0.329/0.329/0.000 ms
+root@sonic:/home/cisco# 
+
+```
+#### Pass/Fail Criteria
+ * Verfiy ping is not going through after removing pcie link.
+ * Verify ping works between dpu and npu after bringing back up the link
+
+### 1.11 Check graceful restart of NPU
+
+#### Steps
+ * Use command `reboot` to get reboot the host
+ * Wait for NPU to come up.
+ * Use command `show interface status` to chece the interfaces are up
+
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch:
+
+root@sonic:/home/cisco# reboot
+.
+.
+.
+.
+root@sonic:/home/cisco# show interface status
+```
+#### Pass/Fail Criteria
+ * Verfiy all the interface are up.
+
+### 1.12 Check the NTP date and timezone between DPU and NPU
+
+#### Steps
+ * Use command `date` to get date and time zone on host.
+ * Use command `ssh admin@169.254.x.x` to enter into required dpu.
+ * Use command `date` again on dpu to show date and time zone of dpu.
+   
+#### Verify in
+ * Switch and dpu
+   
+#### Sample Output
+```
+On Switch:
+
+root@sonic:/home/cisco# date
+.
+.
+root@sonic:/home/cisco# ssh admin@169.254.24.1
+
+.
+.
+
+On DPU:
+root@sonic:/home/cisco# date
+```
+#### Pass/Fail Criteria
+ * Verfiy both the date and time zone are same
+
+### 1.13 Check the Health of Switch and DPUs
+
+#### Steps
+ * Use command `show chassis health-events` to get chassis health events. 
+   
+#### Verify in
+ * Switch and dpu
+   
+#### Sample Output
+```
+On Switch:
+
+root@sonic:/home/cisco# show chassis health events
+
+```
+#### Pass/Fail Criteria
+ * Verfiy that the output is showing only errors related to given option.
+
+### 1.14 Check memory on Host
+
+#### Steps
+ *  Use command `TOP` to show process and memory it is using
+   
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch:
+
+root@sonic:/home/cisco# top
+```
+
+#### Pass/Fail Criteria 
+ * Verfiy the CPU within a threshold value on DPU
```

Comment:
typo

Reply:
Are we providing this threshold and if not what determines it?

Reply:
Also, this is specified for the switch, please adjust to replace DPU w/ switch

Reply:
corrected typo. dpu is separate case